packages/react-components/react-portal/stories/toMountNodeProps/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import descriptionMd from './toMountNodePropsDescription.md';
+
+export { Default } from './Default.stories';
+
+export default {
+  title: 'Utilities/Portals/toMountNodeProps',
```

Comment:
I think this will be easier to find under Components/Portal

Reply:
Makes sense, moved:

<img width="252" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/05c8910d-588a-41fb-b9cc-9b0a29b66b19">